node-registrar/cmds/main.go

```diff
@@ -0,0 +1,74 @@
+package cmds
+
```

Comment:
this file should be registar.go and should have the main func

func main() {

    ...
    if err:=Run() {
        log.Fatal(....)
    }
}
and change the Run function to return an err